### PR TITLE
Document how to update SQLite source dependencies.

### PR DESCRIPTION
**Pull request by Mariatta**
Update the README with the instructions.
Use ReST.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+3. Download SQLite source from ``sqlite.org``.
+
+4. Unzip it to the checked out branch on step 2.
+
+5. Commit and push the changes.
+
+6. Create the PR, with ``sqlite`` as the base branch.
+
+Once the PR has been merged, tag the commit as ``sqlite-<full version specified in CPython's PCbuild/get_externals.bat>``.
```
Comment (zware) [inline README.rst]: s/specified/to be used/

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Source for packages that the CPython build process depends on.
+
+It is currently expected that this will only be useful on Windows,
+and in any case you should never need to clone this repository
+unless you are updating its contents.
+
+.. contents::
+
+Updating SQLite Source Dependencies
```
Comment (zware) [inline README.rst]: I think we can generalize this.  Most dependencies (sqlite, bzip2, xz, zlib) follow the same instructions; tcl and tk also follow the same instructions, but have an extra step afterwards that @zooba has to do (updating [cpython-bin-deps](../../../cpython-bin-deps/)); openssl will have a much-expanded step 5.  Maybe leave the steps below as is, but note that `sqlite` is just used an an example, and the same steps apply to the ones I listed above?

Comment (Mariatta) [reply README.rst]: Thanks :) so I tried my best at explaining that the other dependencies can be updated in similar fashion. Hope the latest change makes sense..

File: README.rst [hunk 3]
```diff
@@ -19,13 +23,32 @@ Updating SQLite Source Dependencies
...
+Once the PR has been merged, tag the commit as ``sqlite-<full version to be used in CPython's PCbuild/get_externals.bat>``.
+
+For updating ``sqlite``, ``bzip2``, ``xz``, and ``zlib``, follow the above
+instructions.
+
+For ``tcl`` and ``tk``, after step 5, `cpython-bin-deps
+<https://github.com/python/cpython-bin-deps/>`_
+needs to be updated.
+
+For ``openssl``, step 5 will be more complicated :)
```
Comment (zware) [inline README.rst:43]: For each of these, we can link to the source site:

sqlite: https://www.sqlite.org/download.html
bzip2: http://www.bzip.org/downloads.html
xz: https://tukaani.org/xz/
zlib: https://zlib.net/
tcl/tk: https://tcl.tk/software/tcltk/download.html
openssl: https://www.openssl.org/source/

File: README.rst [hunk 4]
```diff
@@ -19,13 +23,32 @@ Updating SQLite Source Dependencies
 
       git checkout -b <branch-name>-sqlite upstream/sqlite
 
-3. Download SQLite source from ``sqlite.org``.
+3. Download SQLite source from `sqlite.org <https://www.sqlite.org>`_.
 
 4. Unzip it to the checked out branch on step 2.
```
Comment (zware) [inline README.rst]: s/checked out branch on/branch checked out in/

File: README.rst [hunk 5]
```diff
@@ -19,13 +23,32 @@ Updating SQLite Source Dependencies
 
       git checkout -b <branch-name>-sqlite upstream/sqlite
 
-3. Download SQLite source from ``sqlite.org``.
+3. Download SQLite source from `sqlite.org <https://www.sqlite.org>`_.
```
Comment (zware) [inline README.rst]: We should add checking the hash to this step.

File: README.rst (hunk 3 above)
Comment (Mariatta) [reply README.rst:43]: Thanks, I added a "Download links" section listing the above.

**Comment by zware** [comment]
Thanks @Mariatta :)

**Pull request merged**